Directory.Build.targets

```diff
@@ -156,25 +154,27 @@ Expected to exist:
   <!-- Build the iOS SDK: dotnet msbuild /t:BuildiOSSDK src/Sentry.Unity -->
   <Target Name="BuildiOSSDK" Condition="$([MSBuild]::IsOSPlatform('OSX'))
           And '$(MSBuildProjectName)' == 'Sentry.Unity'
-          And (!Exists('$(SentryiOSDeviceArtifactsDestination)') OR !Exists('$(SentryiOSSimulatorArtifactsDestination)'))" BeforeTargets="BeforeBuild">
+          And (!Exists('$(SentryiOSArtifactsDestination)'))" BeforeTargets="BeforeBuild">
     <Error Condition="!Exists('$(SentryCocoaRoot)')" Text="Couldn't find the Cocoa root at $(SentryCocoaRoot)."></Error>
     <Message Importance="High" Text="Building Sentry iOS SDK."></Message>
 
-    <Exec WorkingDirectory="$(SentryCocoaRoot)" Command="carthage build --use-xcframeworks --no-skip-current --platform iOS"></Exec>
+    <!--    Workaround to enable building the Cocoa SDK with Bitcode enabled -->
+    <Message Importance="High" Text="Creating xcconfig file to enable Bitcode." />
+    <WriteLinesToFile File="$(SentryCocoaRoot)unity.xcconfig" Lines="ENABLE_BITCODE = YES" Overwrite="true" />
```

Comment:
what happens if we have it enabled here and users want to disable it (in Unity) - would everything work normally? Just checking because nothing coming from xcode would surprise me

Reply:
Yes. I did test that and Xcode just rolls with it.